packages/components/src/autocomplete/index.js

```diff
@@ -269,6 +269,9 @@ function useAutocomplete( {
 		event.preventDefault();
 	}
 
+	// textContent is a primitive (string), memoizing is not strictly necessary
+	// but since the autocompleter is a potential bottleneck for the editor type metric
+	// it's over optimized.
```

Comment:
Should we mention that this is preemptive? `Over optimized` makes me think that is definitely needed.